README.md

```diff
@@ -4,68 +4,95 @@ languages:
 - csharp
 products:
 - azure
+- azure-storage
+- dotnet-core
 description: "This repo contains sample code demonstrating the backup/restore and recoverable deletion functionality of Azure Key Vault using the Azure .NET SDK."
-urlFragment: net-sdk-samples
+urlFragment: key-vault-dotnet-recovery
```

Comment:
This will result in broken links. `urlFragment` should neve be changed. It's unfortunate it was generic before, but should remain the same. Just be more specific going forward.